k8s/yaml/dcgm-on-gke/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why is this file been added?

Reply:
this was a commit from a stale branch, remove now in